packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -1,36 +1,36 @@
+The only flows that support refresh tokens are the resource owner password flow and the authorization code flow. This means that the following combinations of grant type and scope, when sent to `/token` endpoint, return a refresh token:
 
 | Grant Type           | Scope                       |
 | -----------          | -----                       |
 | `authorization_code` | `offline_access` (see Note) |
 | `refresh_token`      | `offline_access`            |
 | `password`           | `offline_access`            |
 
-> **Note:** The authorization code flow is unique, in that the `offline_access` scope has to be requested as part of the code request to the `/authorize` endpoint, and not the request sent to the `/token` endpoint.
+> **Note:** The authorization code flow is unique, in that the `offline_access` scope must be requested as part of the code request to the `/authorize` endpoint and not the request sent to the `/token` endpoint.
 
-### Get a Refresh Token with the Code Flow
+### Get a refresh token with the code flow
 
-In the case of the Authorization Code flow, you use the Authorization Server's `/authorize` endpoint to get an authorization code, specifying an `offline_access` scope. You then send this code to the `/token` endpoint to get an access token and a refresh token.
+In the case of the authorization code flow, you use the Authorization Server's `/authorize` endpoint to get an authorization code, specifying an `offline_access` scope. You then send this code to the `/token` endpoint to get an access token and a refresh token.
 
-> **Note:**  Authorization code with PKCE requests will not return refresh tokens if they are sent from SPAs or other browser-based apps.
+> **Note:**  Authorization code with PKCE requests don't return refresh tokens if they are sent from SPAs or other browser-based apps. Instead, you can [silently refresh tokens](#get-a-new-access-token-id-token-silently-for-your-spa) by making a call to the `/authorize` endpoint.
 
-For more information about this endpoint, see [Obtain an Authorization Grant from a User](/docs/reference/api/oidc/#authorize). For more information about the Authorization Code flow, see [Implementing the Authorization Code Flow](/docs/guides/implement-auth-code/).
+For more information about this endpoint, see [Obtain an authorization grant from a User](/docs/reference/api/oidc/#authorize). For more information about the authorization code flow, see [Implementing the authorization code flow](/docs/guides/implement-auth-code/).
 
-### Get a Refresh Token with the Resource Owner Password Flow
+### Get a refresh token with the resource owner password flow
 
-For the Resource Owner Password flow, you use the Authorization Server's `/token` endpoint directly.
+For the resource owner password flow, you use the authorization server's `/token` endpoint directly.
```

Comment:
This capitalization doesn't match what you have on line 19. It's either "Authorization Server's" or "authorization server's"